### ISSUE TITLE
[FPS] Run Multiple Proposals with Single Command

### ISSUE THREAD

**Issue by ElliotFriedman** [ethereum-optimism/superchain-ops#432]
Create a config file system and runner file that allows running multiple tasks back to back to simulate multiple tasks where task `B` depends on task `A`.

This task runner should smoothly integrate with the integration tests, allowing the monorepo tests to be run against the latest tasks in the repository.

**Comment by sebastianst** [comment]
duplicate of https://github.com/ethereum-optimism/superchain-ops/issues/342 ?

**Comment by mds1** [comment]
Closing as implemented/duplicate. #342 was closed by #819, and current main supports stacked simulations via `just simulate-stack` / `just list-stack`.
